src/lib.rs

```diff
@@ -49,34 +61,25 @@ impl Drone for MyDrone {
             packet_recv,
             pdr,
             packet_send,
-            // TODO: I don't understand why we need to use RefCell here???
-            known_flood_ids: RefCell::new(HashSet::new()),
+            known_flood_ids: HashSet::new(),
```

Comment:
I guess we didn't need it haha

Reply:
Yes, I just made every function to take an `&mut self` instead of an `&self` to avoid using RefCell